### PR TITLE
Update Play

### PR DESCRIPTION
**Pull request by fredex42**
## What does this change?

- Updates Play framework
- Updates Native Packager
- Remove un-needed dependency overrides

## How to test

If it runs, we are good 😀 

## How can we measure success?

No more vulns

## Have we considered potential risks?

n/a

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,9 +1,8 @@
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.21")
+addSbtPlugin("org.playframework" % "sbt-plugin"            % "3.0.2")
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"          % "2.0.0")
-addSbtPlugin("com.typesafe.sbt"  % "sbt-twirl"             % "1.5.1")
-addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.9")
+
+addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.16")
+addDependencyTreePlugin
 
 //this is required because scala compiler depends on a higher version of scala-xml than sbt-native-packager dependencies
```
Comment (codingWithRach) [inline project/plugins.sbt:7]: Maybe delete the comment as well as the code?

**Review by codingWithRach — APPROVED** [review]
Looks good - but a suggestion to remove a comment that has become superfluous

**Review by JustinPinner — APPROVED** [review]
Cleaned out the old, stuck reindexes, redeployed `main`, re-ran a one day reindex successfully, then repeated with this branch and it worked as expected so we can approve this now 👍

File: project/plugins.sbt (hunk 1 above)
Comment (fredex42) [reply project/plugins.sbt:7]: Thanks for spotting 👀

**Pull request merged**